### ISSUE TITLE
[a11y] Cannot create a new App Resource

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1181]
**Describe the bug**
When attempting to create a new App Resource under App Resources, the following error occurs:

```
Unexpected Error

An unexpected error has occurred
Please reload the page and try again.
If this issue persists, please contact your IT support or if this is a Specify Cloud database, contact [support@specifysoftware.org](mailto:support@specifysoftware.org)

Error Message
Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

    at div
    at div
    at d (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:1934730)
    at E (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:1925794)
    at div
    at e (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:1957314)
    at e (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:1951723)
    at h (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:47586)
    at https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:50385
    at c (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:51405)
    at E (https://a11y.test.specifysystems.org/static/js/main.781e9f8a90c6ac8ca79e.bundle.js:2:26468)
```

**To Reproduce**
1. Go to App Resources (/specify/appresources)
2. Attempt to create a new app resource
3. See error

**Expected behavior**
This should not happen.

**Desktop (please complete the following information):**
 - Windows 10, Edge

**Comment by maxxxxxdlp** [comment]
It looks like the original bug has already been fixed, but a new bug occurred, preventing this same action 😂 
Should be fixed now

**Comment by grantfitzsimmons** [comment]
There is a new issue going on now for me!

When creating a new app resource, it does not populate the list of resources on the left.

See `/specify/appresources/146/` on fresh fish.

It does not appear in the left menu but it exists.

**Comment by maxxxxxdlp** [comment]
Please create a separate issue out of it

**Comment by grantfitzsimmons** [comment]
https://github.com/specify/specify7/issues/1265

The app resource is saved, just not visible.
